Utilities/Reload-Module/Reload-Module.psm1

```diff
@@ -11,9 +11,11 @@
     Reload-Module "C:\Path\To\PSModules"
 #>
 
-
 function Reload-Module
 {
+    [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute(
+        'PSUseApprovedVerbs',
+        Justification = 'Use distinctive name to confusion with existing cmdlets such as Import-Module.')]
```

Comment:
```suggestion
        Justification = 'Use distinctive name to avoid confusion with existing cmdlets such as Import-Module.')]
```

Reply:
Assuming we want to _avoid_ confusion
![image](https://user-images.githubusercontent.com/4007293/179867496-dbdfe30f-7132-4f5c-a2ba-69cd1e27fcd6.png)